src/Client/Features/FreeAgents/List.razor

```diff
@@ -25,8 +25,8 @@
                 <MudTh><MudTableSortLabel SortBy="new Func<FreeAgentItem,object>(x=> x.Name)">Name</MudTableSortLabel></MudTh>
                 <MudTh><MudTableSortLabel SortBy="new Func<FreeAgentItem,object>(x=> x.Position)">Position</MudTableSortLabel></MudTh>
                 <MudTh><MudTableSortLabel SortBy="new Func<FreeAgentItem,object>(x=> x.Team)">Former Team</MudTableSortLabel></MudTh>
-                <MudTh><MudTableSortLabel SortBy="new Func<FreeAgentItem,object>(x=> x.BiddingEnds)">Bidding Ends</MudTableSortLabel></MudTh>
-                <MudTh><MudTableSortLabel SortBy="new Func<FreeAgentItem,object>(x=> x.HighestBid)">Highest Bid</MudTableSortLabel></MudTh>
+                <MudTh><MudTableSortLabel SortBy="new Func<FreeAgentItem,object>(x=> x.BiddingEndDate)">Bidding Ends</MudTableSortLabel></MudTh>
+                <MudTh><MudTableSortLabel SortBy="new Func<FreeAgentItem,object>(x=> x.HighestBidValue)">Highest Bid</MudTableSortLabel></MudTh>
```

Comment:
“Value” doesn’t add anything to me IMO, thoughts?

Reply:
oops thgought i deleted this